woocommerce/woocommerce-ios
# Update default store after successfully installing and activating Jetpack-the-plugin

Closes: #5576 


### Description


After installing and activating Jetpack-the-plugin, the JCP UI (Jetpack benefits banner, Settings > Install Jetpack) is still shown for the previous JCP site. This PR updates `StoresManager.sessionManager.defaultSite` so that the UI can be updated after Jetpack-the-plugin becomes active.

### Testing instructions


Prerequisite: the account is connected to at least a JCP site (setup tips in p1636091588189400-slack-C6H8C3G23).

- Launch the app and select the Jetpack benefit banner
- Proceed to install Jetpack. Notice that installation and activation steps still succeed. The process will be stuck at "Connecting your store" due to issue mentioned in p91TBi-6EK-p2 (this issue might have been fixed at the time of review)
- Dismiss the screen and navigate back to the Dashboard screen. Repeat steps 1 and 2. Notice that installation and activation steps are done immediately (since the plugin has been installed and activated earlier)
- Tap "Done" after all the steps are complete --> the Jetpack benefits banner should not be shown, and you should not see the row in Settings > Install Jetpack

---
